File: src/test/java/laddergame/domain/PlayerGeneratorTest.java
package laddergame.domain;

import static java.util.Arrays.asList;
import static org.assertj.core.api.Assertions.assertThat;

import org.junit.Test;

public class PlayerGeneratorTest {

	@Test
	public void 콤마로_플레이어_생성() {
		final String player1 = "pobi";
		final String player2 = "honux";
		final String player3 = "crong";
		final String player4 = "jk";
		final String rawPlayers = String.join(",", player1, player2, player3, player4);

		assertThat(PlayerGenerator.generate(rawPlayers))
				.extracting("name").isEqualTo(asList(player1, player2, player3, player4));
	}
}

File: src/test/kotlin/ladder/domain/generators/PlayerGeneratorTest.kt
package ladder.domain.generators

import ladder.domain.PLAYERS
import ladder.domain.Player
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class PlayerGeneratorTest {
    @Test
    fun `플레이어 생성이 정상적으로 되었는지 확인`() {
        val players = PlayerGenerator.generator("Dave,Lee,Kim")
        assertThat(players).isEqualTo(PLAYERS)
    }
}
